### ISSUE TITLE
[CONSIDER] Limit # of PDFs per section

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#17362]
## Description
VHA will be remediating existing PDFs as part of Accessibility work (3rd wave). Editors will be asked to review existing PDFs (in Drupal and Teamsite) and determine the most appropriate format for those which should be retained. Editors will receive training and guidance to help them choose the best location (Drupal or Wordpress) and format for existing and new content. 

How might we limit the number of PDFs permitted per section while allowing flexibility for any future "Thou shalt" requirements in the future? 

We do not yet knw if this will be needed - this is to determine if this type of limitation is possible and rough level of effort.

## Tasks
Evaluate options for limiting the number of PDFs with 
- [ ] feasibility
- [ ] pros/cons/risks 
- [ ] rough estimate for effort

**Comment by xiongjaneg** [comment]
CMS Team is doing lead work on PDFs, Facilities will support as needed
